presentation/src/main/java/utils/OutlinedTextView.kt

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import com.d101.presentation.R
+
+class OutlinedTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+) : androidx.appcompat.widget.AppCompatTextView(context, attrs, defStyleAttr) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line